src/App.tsx

```diff
@@ -1,35 +1,107 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const getBiggestId = () => Math.max(...todos.map(todo => todo.id));
```

Comment:
The `getBiggestId` function may return `-Infinity` if the `todos` array is empty, which would cause an error when adding a new todo. Consider handling the case where `todos` is empty by returning `0` as the default biggest ID.